Localization/TouchDownCheckinScript.ps1

```diff
@@ -67,12 +67,20 @@ Function InitializeRepoForCheckin
 Function CheckinFilesIntoRepo
 {
     #Commit the changes
-    $Argument = 'commit  -m "Latest localized files from touchdown"'
+    $Argument = 'commit -m "Latest localized files from touchdown"'
     ProcessStart $git $Argument $repoPath
 
-    #Push the Changes to the  git server you still need to merge the changes
-    $Argument = "push -u origin " + $TempLocBranch
-    ProcessStart $git $Argument $repoPath
+    #Push the Changes to the git server you still need to merge the changes
+    if ($AuthToken -eq "") {
+        #Unauthorized
+        $Argument = "push origin " + $TempLocBranch
+        ProcessStart $git $Argument $repoPath
```

Comment:
same comment as [this](https://github.com/Microsoft/mobile-center-sdk-android/pull/519#discussion_r141967161).